generated/nifake/attributes.py

```diff
@@ -66,7 +66,7 @@ def __get__(self, session, session_type):
 
     def __set__(self, session, value):
         if type(value) is not self._attribute_type:
```

Comment:
I think this is wrong. @texasaggie97 shouldn't this be isinstance()? That way it works with subclasses.

Reply:
According to https://docs.python.org/3/library/enum.html#restricted-subclassing-of-enumerations. Subclassing an enumeration is allowed only if the enumeration does not define any members. Given that our enums have members, customers cannot create a subclass of our enums.